lib/config/fileconf.go

```diff
@@ -618,6 +618,11 @@ type SSH struct {
 
 	// BPF is used to configure BPF-based auditing for this node.
 	BPF *BPF `yaml:"enhanced_recording,omitempty"`
+
+	// IdleTimeoutMessage is sent to the client when the inactivity timeout
+	// expires. The empty string implies no message should be sent prior to
+	// disconnection.
+	IdleTimeoutMessage string `yaml:"idle_timeout_message,omitempty"`
```

Comment:
Just curious, is it a requirement that this be a part of ssh node config? Have we considered making it a part of cluster configuration maybe? So the idle message can be changed without updating configuration on all nodes and restarting them, esp. if they have a large fleet?

Reply:
Moved to ClusterNetworkingConfig and updated all the appropriate resources.